### PR TITLE
Request Demo Form

### PR DESCRIPTION
**Pull request by julieg18**

### CODE REVIEW

**Review by iceicery — CHANGES_REQUESTED** [review]
Very impressive work. I like the form works well and beautiful. Every focus, hover, layout are exactly the same as Figma. The details you added make the form very user friendly.  I noticed when I open the form  the Title would shift. It looks fine at developer tool so I guess that is because of my big scroll bar disappear when opening the form. Do you think we need to fix it? Maybe add a width for title would work. What do you think?
image

File: src/components/FormInput/FormInput.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        onClick={clearInputValue}
+        className="form-input__clear-btn"
+      >
+        <Icon icon={clearIcon} widthPixelSizes={[16, 24, 24]} />
+      </button>
+      <span
+        id={`${name}-input-error`}
+        className={`form-input__error ${
+          !inputValidationMessage && 'form-input__error_hidden'
+        } ${isInputValid && 'form-input__error_active'}`}
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/components/FormInput/FormInput.js:90]: That was some previously written code that it no longer needed, thanks for spotting it!

File: src/components/FormPopup/FormPopup.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            {buttonText}
+          </Button>
+        </form>
+      </div>
+    </Popup>
+  );
+}
+
+FormPopup.propTypes = {
+  children: PropTypes.node.isRequired,
```
[the remark this answers is not in the document]
Comment (julieg18) [reply src/components/FormPopup/FormPopup.js:43]: Sure! I'll take off required!

**Review by iceicery — APPROVED** [review]
Looks good! Nice job! Approved!

**Pull request merged**